File: MultiFlex/API/JavaRestApi/src/main/java/org/acme/DTO/WareDto.java
package org.acme.DTO;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class WareDto implements Serializable {
    private Integer id;

    private String name;

    private Integer stock;

    private Integer minAmount;

    private Integer maxAmount;

    //private Set<ShelfDto> fächer;

    private List<Integer> shelf_ids;

    //private List<ShelfDto> shelfs;

    //private String type;
    //private TypeDto type;

}

File: MultiFlex/Frontend/Multiflex.Frontend/MultiFlex.Frontend.WebApp/Models/Dto/WareDto.cs
using System.Collections.Generic;

namespace Multiflex.Frontend.WebApp.Models.Dto
{
    public class WareDto : IdentityModel
    {
        public int maxAmount { get; set; }
        public int minAmount { get; set; }
        public string name { get; set; }
        public List<int> shelf_ids { get; set; }
        public int stock { get; set; }
    }
}
